stake-pool/js/src/utils/program-address.ts

```diff
@@ -28,9 +28,14 @@ export async function findStakeProgramAddress(
   programId: PublicKey,
   voteAccountAddress: PublicKey,
   stakePoolAddress: PublicKey,
+  seed?: BN,
```

Comment:
Hm, does this have to be `BN` here? The seed itself for a stake address is `u32` so `number` is enough.

Reply:
Changed